Common/src/Storage/ISessionTable.cs

```diff
@@ -65,25 +65,14 @@ public interface ISessionTable : IInitializable
                                            CancellationToken                   cancellationToken = default);
 
   /// <summary>
-  ///   Cancel a session
-  /// </summary>
-  /// <param name="sessionId">Id of the session to cancel</param>
-  /// <param name="cancellationToken">Token used to cancel the execution of the method</param>
-  /// <returns>
-  ///   The metadata of the cancelled session
-  /// </returns>
-  Task<SessionData> CancelSessionAsync(string            sessionId,
-                                       CancellationToken cancellationToken = default);
-
-  /// <summary>
-  ///   Delete a session
+  ///   Remove a session
   /// </summary>
   /// <param name="sessionId">Id of the session to delete</param>
   /// <param name="cancellationToken">Token used to cancel the execution of the method</param>
   /// <returns>
   ///   Task representing the asynchronous execution of the method
   /// </returns>
-  Task DeleteSessionAsync(string            sessionId,
+  Task RemoveSessionAsync(string            sessionId,
```

Comment:
I am not sure why you renamed this function.

Reply:
Because it conflicts with the extension method I created.
We have two "delete":
- the extension that sets the status, the date and the TTL
- the one that directly deletes the entry

I think I will remove the last one but I'm still hesitating on whether we should use TTL or synchrone calls to delete entries in the database